File: Programmers_12912/Solution.java
package Programmers_12912;

public class Solution {
	public long solution(int a, int b) {
		long answer = 0;
		
		if(a < b) {
			for(int i=a ; i<=b ; i++ ) {
				answer= answer+i;
			}
		}
		else if (a > b) {
			for(int i=b ; i<=a ; i++ ) {
				answer= answer+i;
			}
		}
		else {
			answer = a;
		}
		return answer;
	}

}

File: Programmers_12912/solution.js
function solution(a, b) {
  var answer = 0;
  if (a <= b) {
    for (var i = a; i <= b; i++) {
      answer = answer + i;
      console.log(answer);
    }
  } else {
    for (var i = a; i >= b; i--) {
      answer = answer + i;
      console.log(answer);
    }
  }
  return answer;
}

solution(5, 3);
